apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/OutpatientClaimTransformerV2Test.java

```diff
@@ -147,6 +147,15 @@ public void shouldSetFinalAction() {
   public void shouldSetBillablePeriod() throws Exception {
     // We just want to make sure it is set
     assertNotNull(eob.getBillablePeriod());
+    Extension extension =
+        eob.getBillablePeriod()
+            .getExtensionByUrl("https://bluebutton.cms.gov/resources/variables/claim_query_cd");
+    Coding valueCoding = (Coding) extension.getValue();
+    assertEquals("Final bill", valueCoding.getDisplay());
+    assertEquals("3", valueCoding.getCode());
+    assertEquals(
+        "https://bluebutton.cms.gov/resources/variables/claim_query_cd", valueCoding.getSystem());
```

Comment:
I don't think this as written can ever fail; if the value coding didn't equal this then the extension wouldnt have been found and we would have blown up on 153. Instead of this check, we should add a assertNotNull(extension) after line 151. (Same in the other places this was done in other files)

Reply:
Added that to this test and the other v2 transformer test.